index.html

```diff
@@ -157,34 +159,34 @@
 						---
 
 						### Head
-						Contains the page title and meta information about the page. Meta information is invisible to the user, but has many purposes, like providing information to search engines.
+						Contains the page title and meta information about the page. Meta information is mostly invisible to the user, but has many purposes, like providing information to search engines.
 
 						---
 
 						### Body
-						Contains the content of the page. Everything in the body is visible to the user.
+						Holds the visible content of the page. This is where you put anything you wish the user to see.
```

Comment:
Plenty of things in the body might _not_ be visible, but very little outside the body would be.